sockets/proc_net_tcp/proc_net_tcp.go

```diff
@@ -81,7 +81,7 @@ func hexToIP(word string, ipv6 bool) (net.IP, error) {
 }
 
 // Parses the /proc/net/tcp file
-func parseProcNetTCP(input io.Reader, ipv6 bool) ([]*sockets.SocketInfo, error) {
+func ParseProcNetTCP(input io.Reader, ipv6 bool, containerId string) ([]*sockets.SocketInfo, error) {
```

Comment:
In Go, public methods start with a capital, private ones with a lowercase letter